test/integration/controller.v2/clustertrainingruntime_controller_test.go

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
Do we need to remove these test as well?

Reply:
Yes, we can remove this.

Reply:
Done.